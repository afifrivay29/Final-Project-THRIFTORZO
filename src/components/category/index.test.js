import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Category from '../../components/category';

it('renders category correctly', () => {
	const tree = renderer
		.create(
			<MemoryRouter>
				<Category category='Hobi' active />
				<Category category='Kendaraan' />
			</MemoryRouter>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
