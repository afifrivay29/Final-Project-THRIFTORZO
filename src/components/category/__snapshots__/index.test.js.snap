// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders product card correctly 1`] = `
Array [
  <button
    className="ant-btn ant-btn-primary ant-btn-lg bg-[#E2D4F0] text-[#3C3C3C] border-0 py-3 px-6 h-12 flex items-center rounded-xl btn-category active mr-4"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    <svg
      className="mr-2"
      fill="none"
      height={24}
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="11"
        cy="11"
        r="8"
      />
      <line
        x1="21"
        x2="16.65"
        y1="21"
        y2="16.65"
      />
    </svg>
    <span>
      Hobi
    </span>
  </button>,
  <button
    className="ant-btn ant-btn-primary ant-btn-lg bg-[#E2D4F0] text-[#3C3C3C] border-0 py-3 px-6 h-12 flex items-center rounded-xl btn-category  mr-4"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    <svg
      className="mr-2"
      fill="none"
      height={24}
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="11"
        cy="11"
        r="8"
      />
      <line
        x1="21"
        x2="16.65"
        y1="21"
        y2="16.65"
      />
    </svg>
    <span>
      Kendaraan
    </span>
  </button>,
]
`;
